src/backend/distributed/commands/multi_copy.c

```diff
@@ -2625,6 +2626,14 @@ ProcessCopyStmt(CopyStmt *copyStatement, char *completionTag)
 			}
 
 			query = (Query *) linitial(queryTreeList);
+
+			/*
+			 * Add a username suffix to prevent other users from reading/writing
+			 * the same file. We do this consistently in all functions that interact
+			 * with task files.
+			 */
```

Comment:
I think we should quote the user name on all places. For example, the following user fails to run repartition queries now ` CREATE user "on'ender";`

Reply:
Oh, is it? As far as I followed, the `ereport(WARNING` check relies on the user name being the same? I'll read the code a bit more.